ui/imports/shared/popups/ProfilePopup.qml

```diff
@@ -32,8 +31,6 @@ StatusModal {
     property string userIcon: ""
     property string text: ""
 
-    readonly property int innerMargin: 20
```

Comment:
Are we sure this isn't used anywhere and could potentially break UI when we remove this?